File: problemas/iniciante/1789/1789.py
def main():
    while True:
        try:
            L = int(input())
            valores = [int(x) for x in input().split(' ')]

            maior = max(valores)
            if maior < 10:
                print(1)

            elif maior< 20:
                print(2)
            else:
                print(3)
        except EOFError:
            break
        
if __name__ == "__main__":
    main()

File: problemas/iniciante/1789/1789.rb
def main
    loop do
        begin
            line = gets
            break if line.nil?
            l = Integer(line)
            valores = gets.split(' ').map(&:to_i)
            
            maior = valores.max
            if maior < 10
                puts 1
            elsif maior < 20
                puts 2
            else
                puts 3
            end
        rescue EOFError
            break
        end
    end
end

main